### DISCUSSION TITLE
How to restart a run status sensor without triggering on historical runs?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18320] [Q&A]
Hi folks, I'm using <https://docs.dagster.io/_apidocs/libraries/dagster-slack#dagster_slack.make_slack_on_run_failure_sensor|make_slack_on_run_failure_sensor>, which is super useful but if I disable it via the UI for a long period of time, then when I re-enable it it will fire alerts for all of the job failures that happened while it was disabled. I'd prefer that it didn't "catch up" when I turn the sensor back on. Is there a way to change this behavior?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1701180634808489?thread_ts=1701180634.808489&cid=C01U954MEER).*

**Comment by smackesey** [comment]
Hi, this isn't currently possible but will be soon: https://github.com/dagster-io/dagster/pull/18348/files

I will stack on that PR to add a param to `make_slack_on_run_failure_sensor` that will cause it to fire only for runs that terminated after the sensor started.
